epix/strax_interface.py

```diff
@@ -0,0 +1,92 @@
+import strax
+import time
+import os
+import epix
+import wfsim
+import numpy as np
+import pandas as pd
+import awkward1 as ak
+
+@strax.takes_config(
+strax.Option('input_file',
+             help='Input Geant4 ROOT file', track=True),
```

Comment:
This is a bit problematic. Now we would get for each GEANT4 file a unique hash does not sound like the thing we want.

Reply:
Yeah, but at the same time we'd like to know what file the data came from